api/v1alpha1/flowcollector_types.go

```diff
@@ -228,10 +228,13 @@ type FlowCollectorKafka struct {
 	// kafka topic to use. It must exist, NetObserv will not create it.
 	Topic string `json:"topic"`
 
-	// tls client configuration. When using TLS, verify that the address matches the Kafka port used for TLS, generally 9093.
-	// Note that, when eBPF agents are used, Kafka certificate needs to be copied in the agent namespace (by default it's netobserv-privileged).
+	// TLS client configuration. When using TLS, verify that the address matches the Kafka port used for TLS, generally 9093.
 	// +optional
 	TLS ClientTLS `json:"tls"`
+
+	// SASL authentication configuration.
```

Comment:
We might want to say "unsupported" here.

Reply:
(note that we support strimzi / amq streams, and strimzi / amq streams supports sasl, so idk if supporting sasl should transitively apply to us)